src/util/Injected.js

```diff
@@ -400,7 +400,15 @@ exports.LoadUtils = () => {
             await window.Store.GroupMetadata.update(chatWid);
             res.groupMetadata = chat.groupMetadata.serialize();
         }
-
+        
+        res.lastMessage = null;
+        if (res.msgs && res.msgs.length) {
+            const lastMessage = window.Store.Msg.get(chat.lastReceivedKey._serialized);
```

Comment:
```suggestion
            const lastMessage = window.Store.Msg.find(a => a.id._serialized == chat.lastReceivedKey._serialized);
```

Reply:
.get is sometimes unreliable

Reply:
this doesn't work for me, causes message serialization error

Reply:
Now?

Reply:
it works stably